dlc-manager/src/manager.rs

```diff
@@ -1131,10 +1149,10 @@ where
     ) -> Result<(), Error> {
         offer_channel.validate(
             &self.secp,
-            REFUND_DELAY,
-            REFUND_DELAY * 2,
-            CET_NSEQUENCE,
-            CET_NSEQUENCE * 2,
+            self.options.refund_delay,
+            self.options.refund_delay * 2,
+            self.options.cet_nsequence,
+            self.options.cet_nsequence * 2,
```

Comment:
So I set about trying to make this change in the same way I did for refund_delay, but it became a huge rats nest. I created a struct called `CETnSequenceWindow`, but I couldn't import it from dlc-manager in dlc-messages because of a circular dependency. Then I tried just using a cet_nsequence_min and cet_nsequence_max, and handing those all the way down, but that also became really messy because that sometimes it expects just the "min" value, and it gets really confusing. 

What do you think?

Reply:
I'm not sure I understand why you need to use that in dlc-messages?

Reply:
Ah, I think I was trying to pass the struct with the min/max values too far down , when I could have just been passing the u32. 

Anyway, do you think it would be good to have a CETnSequenceWindow object with a min / max values, or just leave it as a u32 with default 288?

Reply:
As mentioned above I feel like not having a separate object is simpler as in I think the max value is only required in few places.